src/configuration.ts

```diff
@@ -24,6 +24,9 @@ import {arrayify, objectify, stripComments} from "./utils";
 
 export interface IConfigurationFile {
     extends?: string | string[];
+    linterOptions?: {
```

Comment:
Thinking we don't need this `linterOptions` property, `typeCheck` can reside at the top level